encord/client.py

```diff
@@ -246,15 +252,17 @@ def create_image_group(
             if not os.path.exists(file_path):
                 raise encord.exceptions.EncordException(message="{} does not point to a file.".format(file_path))
         short_names = list(map(os.path.basename, file_paths))
+
         signed_urls = self._querier.basic_getter(SignedImagesURL, uid=short_names)
```

Comment:
The signed URL might expire for images toward the back of the list if the upload process takes too long on the earlier images. We want to generate the signed URL immediately before beginning the upload process

Reply:
Well that's an interesting one - it is a batch request right now to get the signed URLs. Do we really want to now unbatch this request? In any case, this seems to be unrelated with this PR. I'd actually rather make the timeout longer on the URLs.